### DISCUSSION TITLE
Suitability for building a legal assistant bot with botonic

### DISCUSSION THREAD

**Discussion by Zolooooo** [hubtype/botonic#2131] [Q&A]
Hi!

We are a legal tech company planning to develop a legal assistant chatbot. I’d like to know more about the suitability of your framework with our needs. So, please answer the following questions:

1.	Is Mongolian supported? 
2.	Is your framework capable of building a chatbot that answers all types of legal questions. Is there a similar use case? (maybe in different industries?) 
3.	If your framework is capable of developing a legal assistance chatbot, approximately how many Q&As (intent & response) do we need to train so that our chatbot provides 10,000 different responses? We’re planning to collect 10,000 answers to 100,000 legal questions (10 questions for each response). Is this enough data to train this type of chatbot? 
4.	What AI methods are used or what types of algorithms are used? 

Best,
Zolo

**Comment by manuelalferez** [comment]
Hi @Zolooooo, 

Thank you for considering Botonic for your legal assistant chatbot needs. Here are the answers to your questions:

1. Currently, Botonic doesn't support Mongolian, but you can extend it to add support for other languages.
2. Yes, Botonic can build a legal chatbot. We have legal industry customers successfully using our platform.
3. It's difficult to determine the exact number of Q&As needed to provide 10,000 different responses. You can generate a large number of responses using AI language models like [ChatGPT](https://chat.openai.com/), which can generate responses based on prompts or contexts.
4. Botonic uses various AI and machine learning algorithms, including TensorFlow, to provide NLU capabilities for chatbots. You can also implement your own custom NLU model.
